Lib/fontTools/varLib/merger.py

```diff
@@ -1015,15 +1030,17 @@ def buildVarDevTable(store_builder, master_values):
 
 @VariationMerger.merger(ot.CaretValue)
 def merge(merger, self, lst):
-	assert self.Format == 1
+	if self.Format != 1:
+		raise VarLibMergeError(f"CaretValue format {self.Format} unknown.")
```

Comment:
like for anchors, same for the caret (the format 2 uses contour points instead of xy coords). it is unsupported, not unknown